StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElementConfiguration.swift

```diff
@@ -30,6 +30,9 @@ extension EmbeddedPaymentElement {
         /// Configuration related to Link
         public var link: LinkConfiguration = LinkConfiguration()
 
+        /// If `true`, allows PayPay as a payment method. Defaults to `false`.
+        @_spi(STP) public var enablePayPay: Bool = false
+
```

Comment:
Why do we need a client-side feature flag here? We've never used one for any previous LPM we released during beta. PayPay will only appear for merchants who have already enabled it and define it on their intent, so adding this flag seems like unnecessary friction. We should prefer having the server decide which LPMs are available rather than the client.

Reply:
cc @jaynewstrom-stripe can we ensure we do the same on Android?

Reply:
We have a client side feature flag that's off in release builds! https://github.com/stripe/stripe-android/blob/master/stripe-core/src/main/java/com/stripe/android/core/utils/FeatureFlags.kt#L19